src/index.html

```diff
@@ -3,11 +3,296 @@
+            class="header__menu-icon"
+            src="images/menu.svg"
+            alt="Menu icon"
+          >
+        </a>
+      </div>
+
+      <div class="header__content">
+        <h1 class="header__motto">
+          The world shades.
+          <br>
+          Your music shines.
+        </h1>
+      </div>
+    </header>
+
+    <nav id="menu" class="menu">
+      <a href="#home">
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="Bose logo"
+        >
+      </a>
+      <a class="menu__close" href="#home">
+        <img
+          src="images/cross.svg"
+          alt="Menu close"
+        >
+      </a>
+
+      <ul class="menu__list">
+        <li class="menu__item">
+          <a class="menu__link" href="#home">Home</a>
+        </li>
+        <li class="menu__item">
+          <a class="menu__link" href="#recommended">recommended</a>
+        </li>
+        <li class="menu__item">
+          <a class="menu__link" href="#categories">categories</a>
+        </li>
+        <li class="menu__item">
+          <a class="menu__link" href="#buy-reasons">How to buy</a>
+        </li>
+        <li class="menu__item">
+          <a class="menu__link" href="#contact">Contacts</a>
+        </li>
+      </ul>
+
+      <p class="menu__number">+1 234 5555-55-55</p>
+
+      <a class="menu__call" href="tel:+1 234 555-55-55">call to order</a>
+    </nav>
+
+    <main>
+      <section id="recommended" class="recommended">
+        <h1 class="recommended__title">Recommended</h1>
+
+        <div class="recommended__soundbars">
+          <img
+            class="recommended__picture"
+            src="images/photos/soundbars/smart/smart-320.png"
+            alt="Bose portable Smart speaker"
+          >
+
+          <div class="recommended__details">
+            <h3 class="recommended__name">Bose portable Smart speaker</h3>
+
+            <p class="recommended__kind">Smart home</p>
+            <p class="recommended__price">$ 399.00</p>
+          </div>
+
+          <img
+            class="recommended__picture"
+            src="images/photos/soundbars/flex/flex-320.png"
+            alt="SoundLink Flex Bluetooth speaker"
+          >
+
+          <div class="recommended__details">
+            <h3 class="recommended__name">SoundLink Flex Bluetooth speaker</h3>
+
+            <p class="recommended__kind">Portable bluetooth</p>
+            <p class="recommended__price">$ 149.00</p>
+          </div>
+
+          <img
+            class="recommended__picture"
+            src="images/photos/soundbars/color/color-320.png"
+            alt="SoundLink Color Bluetooth speaker II"
+          >
+
+          <div class="recommended__details">
+            <h3 class="recommended__name">SoundLink Color Bluetooth speaker II</h3>
+
+            <p class="recommended__kind">Portable bluetooth</p>
+            <p class="recommended__price">$ 129.00</p>
+          </div>
+        </div>
+      </section>
```

Comment:
There should be an empty line between these two sections, according to mates code styling rules there always should be empty lines between siblings for example:
```html
<div>
  <span>First Child<span/>

  <div class="recommended__details">
    <h3 class="recommended__name">SoundLink Color Bluetooth speaker II</h3>

    <p class="recommended__kind">Portable bluetooth</p>

    <p class="recommended__price">$ 129.00</p>
  </div>

  <span>Second Child<span/>
</div>
```

Style this file accordingly

Reply:
I set style with Darek - to add new lines only for siblings with different tags.